static/char_counter/js/char_counter.js

```diff
@@ -0,0 +1,52 @@
+let LineLengthSettingSingleton = 0;
+
+document.addEventListener('DOMContentLoaded', () => {
+	LineLengthSettingSingleton = getLineLength();
+
+	const inputs = document.querySelectorAll('.char-counter-text-input')
+
+	inputs.forEach((input) => {
+		input.addEventListener('input', e => updateCharCountLabel(e.target));
+	});
+
+	document.querySelector('#id_line_length').addEventListener('change', (e) => {
```

Comment:
I ... don't think so. I believe that's set by Django. Though, now that I think about it, it might be worth adding a custom class to that widget, now that I know how. That way the thing our javascript references is under our control